nexussdk/storages.py

```diff
@@ -162,11 +175,15 @@ def create_external_disk_storage(self, org_label: str, project_label: str, endpo
         if credentials is not None:
             payload["credentials"] = credentials
 
+        if max_file_size is not None:
+            payload["maxFileSize"] = max_file_size
+
         return self.create_(org_label, project_label, payload, storage_id)
 
     def update_disk_storage(self, org_label: str, project_label: str, volume: str,
                             storage_id: str, rev: int, read_permission: Optional[str] = None,
-                            write_permission: Optional[str] = None, default: bool = False) -> Dict:
+                            write_permission: Optional[str] = None, max_file_size: Optional[int] = None,
```

Comment:
long

Reply:
in Python 3.x, the int type is of unlimited length by default. There is therefore no 'long integer' in Python 3 anymore.